packages/edit-widgets/src/components/layout/index.js

```diff
@@ -22,30 +26,45 @@ import Notices from '../notices';
 function Layout( { blockEditorSettings } ) {
 	const [ selectedArea, setSelectedArea ] = useState( null );
 	return (
-		<SlotFillProvider>
-			<DropZoneProvider>
-				<BlockEditorKeyboardShortcuts.Register />
-				<Header />
-				<Sidebar />
-				<Notices />
-				<div
-					className="edit-widgets-layout__content"
-					role="region"
-					aria-label={ __( 'Widgets screen content' ) }
-					tabIndex="-1"
-					onFocus={ () => {
-						setSelectedArea( null );
-					} }
-				>
-					<WidgetAreas
-						selectedArea={ selectedArea }
-						setSelectedArea={ setSelectedArea }
-						blockEditorSettings={ blockEditorSettings }
-					/>
-				</div>
-				<Popover.Slot />
-			</DropZoneProvider>
-		</SlotFillProvider>
+		<>
+			<BlockEditorKeyboardShortcuts.Register />
+			<SlotFillProvider>
+				<DropZoneProvider>
+					<FocusReturnProvider>
+						<EditorSkeleton
+							header={ <Header /> }
+							sidebar={ <Sidebar /> }
+							content={
+								<>
+									<Notices />
```

Comment:
Shouldn't `EditorSkeleton` be also more opinionated about the placement of the `Notices` component?

Reply:
I believe the answer is the same https://github.com/WordPress/gutenberg/pull/20431#discussion_r383840832